vite.config.ts

```diff
@@ -1,11 +1,12 @@
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react";
 import copy from "rollup-plugin-copy";
+import WindiCSS from "vite-plugin-windicss";
 
 // https://vitejs.dev/config/
 export default defineConfig({
   base: "",
-  plugins: [react()],
+  plugins: [react(), WindiCSS()],
```

Comment:
for what do you use this `WindiCSS` plugin?

Reply:
to use windicss on my components